interactions/interactions.md

```diff
@@ -45,33 +45,33 @@ Live collaborative editing is possible in CoCalc.
 If one of your collaborators updates a notebook, the rest can see the changes as they are being made (similar to Google Docs).
 
 ## Creating a shared project
-You can create a common shared project from any of your existing projects. You can think of a shared project to be your private course website for the students with automatic forum and code support. By default everybody (collaborators and students) will have **write** access to the project and its associated files.
+You can create a common shared project for your course. Think of a shared project to be your private course website for the students with automatic forum and code support. By default everybody (collaborators and students) will have **write** access to the project and its associated files.
```

Comment:
"Think of a shared project to be your private course" --> "Think of a shared project as your private course"

Reply:
"access to the project and its" --> "access to all files in the shared project".